code/edgeidentity/src/test/java/com/adobe/marketing/mobile/edge/identity/IdentityItemTests.java

```diff
@@ -161,9 +192,10 @@ public void testIdentityItem_fromData_emptyId() {
 		IdentityItem item = IdentityItem.fromData(map);
 
 		// For backward compatibility, an IdentityItem can contain empty identifiers.
-		assertEquals("", item.getId());
-		assertEquals("loggedOut", item.getAuthenticatedState().getName());
-		assertFalse(item.isPrimary());
+		String expected =
+			"{\n" + "  \"id\": \"\",\n" + "  \"authenticatedState\": \"loggedOut\",\n" + "  \"primary\": false\n" + "}";
```

Comment:
formatting

Reply:
When the id value is empty, SpotlessApply make this json into one line.